responses/__init__.py

```diff
@@ -366,6 +368,8 @@ def __init__(
 
         self.match: "_MatcherIterable" = match
         self.call_count: int = 0
+        if "passthrough" in kwargs:
+            self.passthrough = kwargs["passthrough"]
```

Comment:
Ah, my mistake - I tried that, but then saw some unit test failures.  And now I've realized that the tests were failing since the start.  Taking a look at this again now.

Reply:
I'm not 100% certain why yet, but the change to use a named keyword argument caused problems for call sites using the `PassthroughResponse` directly (resolved by [adding an explicit super-constructor call](https://github.com/getsentry/responses/pull/558/commits/64f32f5c3b853a271cb61889e1b566727bae3edf#diff-27a24757cfb28e26971f5b4f49ba6f533b5a51c770d57369cebbef09c7433d6cR602-R609))